### PR TITLE
fix: icon scss integration & simplified icon reference

### PR DESCRIPTION
**Pull request by mfranzke**
The icon helper was missing a relevant SCSS placeholder, as well as meaningful fallbacks that could make including icons simpler.

And we needed to simplify those styling related declarations only to have them included within SCSS and won't need it within markup at all.

### CODE REVIEW

File: packages/foundations/scss/icon/_icons.helpers.scss [hunk 1]
```diff
@@ -30,10 +31,12 @@
 			// * use !important to prevent issues with browser extensions that change fonts
 			font-family: var(
 				--icon-font-family,
-				"#{"icons-" + $size + "-" + $style}, missing-icons"
+				#{"icons-" + $size + "-" + $style},
+				missing-icons,
+				var(--db-base-icon-font-family)
```
Comment (nmerget) [inline packages/foundations/scss/icon/_icons.helpers.scss:36]: Should this become before missing-icons?

Comment (mfranzke) [reply packages/foundations/scss/icon/_icons.helpers.scss]: I've removed those leftovers.

File: packages/foundations/scss/icon/_icons.helpers.scss [hunk 2]
```diff
@@ -29,11 +30,10 @@
 			/*** icon - partial ***/
 			// * use !important to prevent issues with browser extensions that change fonts
 			font-family: var(
```
Comment (nmerget) [inline packages/foundations/scss/icon/_icons.helpers.scss:32]: 90% of this is included in `%icon` inside `_icons.placeholder`, why do we write it again?

Can we just create a `%icon-partial` which is then included in `%icon` or the other way around?

Comment (mfranzke) [reply packages/foundations/scss/icon/_icons.helpers.scss:32]: yes, that's part of https://github.com/db-ui/mono/pull/509 already. But currently the SCSS Mixin way doesn't work at all, as the placeholder wasn't included previously.

File: __snapshots__/alert/alert.spec.tsx-snapshots/DBAlert-component-should-match-screenshot-for-device-desktop-1-chromium-linux.png
Comment (nmerget) [inline __snapshots__/alert/alert.spec.tsx-snapshots/DBAlert-component-should-match-screenshot-for-device-desktop-1-chromium-linux.png:1]: I guess there is some error here

Comment (mfranzke) [reply __snapshots__/alert/alert.spec.tsx-snapshots/DBAlert-component-should-match-screenshot-for-device-desktop-1-chromium-linux.png:1]: I assume, that this link is incorrectly configured, as internal links get that arrow according to the styleguide.

**Review by nmerget — APPROVED** [review]

**Pull request merged**